src/color.cc

```diff
@@ -15,23 +17,249 @@
 #define WHITESPACE \
   while (' ' == *str) ++str;
 
+#define WHITESPACE_OR_COMMA \
+  while (' ' == *str || ',' == *str) ++str;
+
+
+/*
+ * Parse integer value
+ */
+
+template <typename parsed_t>
+static bool parseInteger(const char** pStr, parsed_t *pParsed)
```

Comment:
ok .. I think I get it .. we're doing _lots_ of parsing and need this to be just a step in that, ignore my comment then